inductiva/molecules/scenarios/utils.py

```diff
@@ -0,0 +1,16 @@
+"""Tools to analyze molecular dynamics simulations."""
+import MDAnalysis as mda
+from MDAnalysis import transformations
+
+
+def unwrap_trajectory(topology, trajectory):
+    """Unwrap the trajectory of a simulation to 
+    remove effects from periodic boundary conditions.
```

Comment:
Sorry for being pedantic, but here specify that the reason you are doing this is just a visualization issue. The way it is worder right now seems like you are changing the trajectory to correct for artifacts caused by the PBC, which is not the case.